<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="50010" started-at="2018-11-02T17:21:28Z" finished-at="2018-11-02T17:22:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="50010" started-at="2018-11-02T17:21:28Z" finished-at="2018-11-02T17:22:18Z">
      <class name="test.java.NGtests">
        <test-method status="PASS" signature="BeforeTests()[pri:0, instance:test.java.NGtests@29774679]" name="BeforeTests" is-config="true" duration-ms="4783" started-at="2018-11-02T17:21:28Z" finished-at="2018-11-02T17:21:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTests -->
        <test-method status="PASS" signature="frequencyOfWrongVsCorrect()[pri:0, instance:test.java.NGtests@29774679]" name="frequencyOfWrongVsCorrect" duration-ms="7850" started-at="2018-11-02T17:21:33Z" finished-at="2018-11-02T17:21:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- frequencyOfWrongVsCorrect -->
        <test-method status="PASS" signature="AfterTests()[pri:0, instance:test.java.NGtests@29774679]" name="AfterTests" is-config="true" duration-ms="767" started-at="2018-11-02T17:21:40Z" finished-at="2018-11-02T17:21:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTests -->
        <test-method status="PASS" signature="BeforeTests()[pri:0, instance:test.java.NGtests@29774679]" name="BeforeTests" is-config="true" duration-ms="4332" started-at="2018-11-02T17:21:41Z" finished-at="2018-11-02T17:21:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTests -->
        <test-method status="PASS" signature="verifyCounterStreak()[pri:0, instance:test.java.NGtests@29774679]" name="verifyCounterStreak" duration-ms="4300" started-at="2018-11-02T17:21:46Z" finished-at="2018-11-02T17:21:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyCounterStreak -->
        <test-method status="PASS" signature="AfterTests()[pri:0, instance:test.java.NGtests@29774679]" name="AfterTests" is-config="true" duration-ms="759" started-at="2018-11-02T17:21:50Z" finished-at="2018-11-02T17:21:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTests -->
        <test-method status="PASS" signature="BeforeTests()[pri:0, instance:test.java.NGtests@29774679]" name="BeforeTests" is-config="true" duration-ms="4042" started-at="2018-11-02T17:21:51Z" finished-at="2018-11-02T17:21:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTests -->
        <test-method status="FAIL" signature="verifyCounterTriesAndCorrect()[pri:0, instance:test.java.NGtests@29774679]" name="verifyCounterTriesAndCorrect" duration-ms="4601" started-at="2018-11-02T17:21:55Z" finished-at="2018-11-02T17:21:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at org.testng.asserts.Assertion$27.doAssert(Assertion.java:347)
at org.testng.asserts.Assertion.executeAssert(Assertion.java:30)
at org.testng.asserts.Assertion.doAssert(Assertion.java:15)
at org.testng.asserts.Assertion.assertEquals(Assertion.java:344)
at pages.GamePage.verifyCounterTriesAndCorrect(GamePage.java:161)
at test.java.NGtests.verifyCounterTriesAndCorrect(NGtests.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyCounterTriesAndCorrect -->
        <test-method status="PASS" signature="AfterTests()[pri:0, instance:test.java.NGtests@29774679]" name="AfterTests" is-config="true" duration-ms="761" started-at="2018-11-02T17:21:59Z" finished-at="2018-11-02T17:22:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTests -->
        <test-method status="PASS" signature="BeforeTests()[pri:0, instance:test.java.NGtests@29774679]" name="BeforeTests" is-config="true" duration-ms="3709" started-at="2018-11-02T17:22:00Z" finished-at="2018-11-02T17:22:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTests -->
        <test-method status="FAIL" signature="verifyWhoIsAndLineupChange()[pri:0, instance:test.java.NGtests@29774679]" name="verifyWhoIsAndLineupChange" duration-ms="4727" started-at="2018-11-02T17:22:04Z" finished-at="2018-11-02T17:22:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
null,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal,
   first and second equal
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at pages.GamePage.verifyWhoIsAndLineupChange(GamePage.java:117)
at test.java.NGtests.verifyWhoIsAndLineupChange(NGtests.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhoIsAndLineupChange -->
        <test-method status="PASS" signature="AfterTests()[pri:0, instance:test.java.NGtests@29774679]" name="AfterTests" is-config="true" duration-ms="771" started-at="2018-11-02T17:22:08Z" finished-at="2018-11-02T17:22:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTests -->
        <test-method status="PASS" signature="BeforeTests()[pri:0, instance:test.java.NGtests@29774679]" name="BeforeTests" is-config="true" duration-ms="3686" started-at="2018-11-02T17:22:09Z" finished-at="2018-11-02T17:22:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTests -->
        <test-method status="PASS" signature="verifyWhoIsAppearsInLineup()[pri:0, instance:test.java.NGtests@29774679]" name="verifyWhoIsAppearsInLineup" duration-ms="4105" started-at="2018-11-02T17:22:13Z" finished-at="2018-11-02T17:22:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhoIsAppearsInLineup -->
        <test-method status="PASS" signature="AfterTests()[pri:0, instance:test.java.NGtests@29774679]" name="AfterTests" is-config="true" duration-ms="775" started-at="2018-11-02T17:22:17Z" finished-at="2018-11-02T17:22:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTests -->
      </class> <!-- test.java.NGtests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
